src/nccl_ofi_rdma.cpp

```diff
@@ -137,15 +132,15 @@ static inline int free_base_req(uint64_t *num_inflight_reqs,
 static inline int check_post_rx_buff_req(nccl_net_ofi_rdma_req_t *rx_buff_req);
 
 
-static nccl_net_ofi_rdma_domain_t *rdma_endpoint_get_domain(nccl_net_ofi_rdma_ep_t *ep)
+inline nccl_net_ofi_rdma_domain_t *nccl_net_ofi_rdma_ep_t::rdma_endpoint_get_domain()
```

Comment:
shouldn't need the inline keyword here?

Reply:
In latest force push, I updated endpoint member functions so that only functions defined in the class declaration (e.g. getters) are inline, and all functions defined in the source cpp files are not inline.